src/foam/util/SafetyUtil.java

```diff
@@ -27,6 +27,12 @@ public static boolean equals(Object o1, Object o2) {
     return compare(o1, o2) == 0;
   }
 
+  public static boolean equalsIgnoreCase(String o1, String o2) {
+    if ( o1 == o2 )  return true;
+    if ( o1 == null || o2 == null ) return false;
+    return o1.compareToIgnoreCase(o2) == 0;
```

Comment:
Why not just call equalsIgnoreCase()?

Reply:
Somehow I actually forgot about java.String's function `equalsIgnoreCase()` at that moment
how's it now?